python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,99 +3,285 @@
+            'libnsl.a',
+            'libpthreads.a',
+            'libpthreads_compat.a',
+            'libssl.a',
+            'libtli.a',
+            'libz.a',
+            ]
+        # sys.platform could be 'aix5', 'aix6' etc.
+        aix_version = int(sys.platform[-1])
+        if aix_version >= 7:
+            allowed_deps.extend([
+                'libthread.a',
+            ])
+    elif platform_system == 'sunos':
+        # This is the standard list of deps for a Solaris 10 build.
+        # For now, we include the major versions for Solaris libs.
+        allowed_deps = [
+            'libaio.so.1',
+            'libc.so.1',
+            'libcrypt_i.so.1',
+            'libcrypto.so.0.9.7',
+            'libcrypto_extra.so.0.9.7',
+            'libdl.so.1',
+            'libdoor.so.1',
+            'libgen.so.1',
+            'libintl.so.1',
+            'libm.so.2',
+            'libmd.so.1',
+            'libmp.so.2',
+            'libnsl.so.1',
+            'librt.so.1',
+            'libscf.so.1',
+            'libsocket.so.1',
+            'libsqlite3.so.0',
+            'libssl.so.0.9.7',
+            'libssl_extra.so.0.9.7',
+            'libuutil.so.1',
+            'libz.so.1',
+            ]
+    elif platform_system == 'darwin':
+        # This is the minimum list of deps for OS X.
+        allowed_deps = [
+            'ApplicationServices.framework/Versions/A/ApplicationServices',
+            'Carbon.framework/Versions/A/Carbon',
+            'CoreFoundation.framework/Versions/A/CoreFoundation',
+            'CoreServices.framework/Versions/A/CoreServices',
+            'SystemConfiguration.framework/Versions/A/SystemConfiguration',
+            'libSystem.B.dylib',
+            'libcrypto.0.9.8.dylib',
+            'libgcc_s.1.dylib',
+            'libssl.0.9.8.dylib',
+            'libz.1.dylib',
+            ]
+    return allowed_deps
+
+
+def get_actual_deps():
+    """
+    Return the list of actual deps for the newly-built binaries.
+    """
     try:
-        from ctypes import windll
-        windll
+        raw_deps = subprocess.check_output(script_helper).splitlines()
     except:
-        print '"ctypes - windll" missing.'
-        exit_code = 1
+        sys.stderr.write('Could not get the deps for the new binaries.\n')
+    else:
+        libs_deps = []
+        for line in raw_deps:
+            # We exclude any occuring lines listing the examined files as we
+            # know they start with './' (this is a problem in AIX and OS X).
+            if line[:2] != './' :
+                libs_deps.append(line.split()[0])
+        return list(set(libs_deps))
+
+
+def get_unwanted_deps(allowed_deps, actual_deps):
+    """
+    Return unwanted deps for the newly-built binaries.
+    allowed_deps are hardcoded deps for binaries built for the current OS.
+    actual_deps are the deps found for the newly-built binaries.
+    """
+    # Check actual deps one by one to see if there is a substring of each of
```

Comment:
As said above, I'm a bit confused. In #13 you said: _[…] doctrings. Keep it simple. Don't add details about implementation. The docstring is there for the people using this method. It should state what it does and the format of the returned value._

I have no opinion on that whatsoever, but by looking to the style guide and PEPs (especially PEP 257), I think I shouldn't add that comment. Following those docs, I have added only the lines for the return value and the parameters. But then, the PEPs enforce things you haven't asked for, so I'm not sure.

Please clarify this for me. Thanks!